pyproject.toml

```
@@ -90,9 +86,7 @@ djangorestframework = "~3.13.1"
 # Swagger schema generation for the REST API
 drf-yasg = {extras = ["validation"], version = "~1.20.0"}
 # Git integrations for Python
-# NOTE: GitPython 3.1.23 and later drop support for Python 3.6
-# NOTE: GitPython 3.1.19 through 3.1.23 are yanked from PyPI
-GitPython = "3.1.18"
+GitPython = "3.1.27"
```

```suggestion
GitPython = "~3.1.27"
```

May need to `poetry add GitPython~3.1.27` to get pyproject also updated